package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "email-alert-frontend",
+  "version": "1.0.0",
+  "description": "A frontend for creating and managing email subscriptions.",
```

Comment:
Somewhat moot given this is private but should reference GOV.UK in the description.